### ISSUE TITLE
Optimization of the code 

### ISSUE THREAD

**Issue by Akshat111111** [Akshat111111/Hedging-of-Financial-Derivatives#2]
This code can be optimized by integrating more technical indicators or other concepts of Machine learning or deep learning

**Comment by shubha987** [comment]
@Akshat111111 hey I can try this part ...if possible assign me this issue

**Comment by Akshat111111** [comment]
I have assigned this issue to you.Can you please share your thought process of what you are thinking to implement?? or which Machine learning or deep learing algorithm?? or Something else??

**Comment by shubha987** [comment]
Well I have just started machine learning algorithms lessons but will adapt with it so it will take me some time....if you have any suggestions for me please give

**Comment by Akshat111111** [comment]
Yeah,no worries.Learn the ML at your own pace.I would suggest you to focus more on Random forest and Decision tree.

**Comment by shubha987** [comment]
okay

**Comment by Akshat111111** [comment]
Hey @shubha987 are you working on it ??

**Comment by shubha987** [comment]
Actually I am still learning it . You can unassign me but I will surely work on it irrespective of kwoc

**Comment by Akshat111111** [comment]
No worries,keep learning,as learning is the main priority.You are welcome to contribute in it anytime

**Comment by apooyadv** [comment]
You can assign me this, I will use try and test different ml algos , use different activation functions and maybe hypertune paramteres if need be.

**Comment by Akshat111111** [comment]
Sure,you can move forward with it

**Comment by Celaena24** [comment]
Can I also be assigned to this issue? I am new to ML but I have worked with Decision Trees before

**Comment by Akshat111111** [comment]
@Celaena24 I will suggest you to make a new issue targetting decision tree implementation

**Comment by Akshat111111** [comment]
@apooyadv are you working on it??

**Comment by apooyadv** [comment]
yes, I am. I just have to write more efficient code for each of the models right?

**Comment by Akshat111111** [comment]
> yes, I am. I just have to write more efficient code for each of the models right?

yeah and make it more organized.
